orm/models.py

```diff
@@ -94,14 +94,16 @@ def build_select_expression(self):
 
         return expr
 
-    def filter(self, **kwargs):
+    def filter(self, exclude=False, **kwargs):
```

Comment:
I wonder if we should use `_exclude`, to make it private-ish, and help ensure it shouldn't clash with any of the provided `kwargs`?

Reply:
Totally true, I isolated the parsing logic in a private method avoiding any posible clash with field names